pyproject.toml

```diff
@@ -33,9 +33,19 @@ python = ">=3.7,<4"
 asyncio-dgram = "2.1.2"
 dnspython = "2.2.1"
 
-[tool.poetry.dev-dependencies]
-black = "^22.8.0"
+[tool.poetry.group.dev.dependencies]
+pre-commit = "^2.20.0"
+tox = "^3.26.0"
+tox-poetry = "^0.4.1"
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.1.3"
+pytest-asyncio = "^0.19.0"
+pytest-cov = "^3.0.0"
 coverage = "^6.4"
+
+[tool.poetry.group.lint.dependencies]
```

Comment:
So why not `style`?

Reply:
Because more than just "style" is checked by this group of tools that also includes linters. Most projects have a "lint" command to run this group of tools. Very popular amongst nodejs projects too. Sometimes `format` is split out into its own command too, but usually still included in the `lint` command.